### ISSUE TITLE
azure-cosmos-db-emulator-docker unable to connect mongo client

### ISSUE THREAD

**Issue by AnunnakiSelva** [Azure/azure-cosmos-db-emulator-docker#33]
I can able to install the docker container in my local , But when trying to connect with Mongo 3T or with my .net application which uses mongo client nuget package. It throws timeout error .  

> "A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "172.29.117.101:10255" }", EndPoint: "172.29.117.101:10255", State: "Disconnected", Type: "Unknown", LastUpdateTimestamp: "2019-08-28T11:48:28.0587550Z" }] }."

Only I have problem with docker container . when i do the same with  Cosmos Db emulator everything works fine in local. 
I have tried connecting  with azure document db it works, But facing problem  when i trying the same with mongo api  which runs inside the docker container .

Basically Am trying to run integration test in my azure pipeline by adding azure cosmos db emulator task, which unfortunately not work with mongo api alone .

**Comment by fumasa** [comment]
I'm facing the same problem... trying to run an container with **/EnableMongoDbEndpoint** didn't make any difference... I still can see the port being listening by **netstat** inside the container.

**Comment by niteshvijay1995** [comment]
Add `&tlsInsecure=true` in mongo connection string. 
Example - 
`mongodb://localhost:C2y6yDjf5%2FR%2Bob0N8A7Cgv30VRDJIWEHLM%2B4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw%2FJw%3D%3D@localhost:10255/admin?ssl=true&retrywrites=false&tlsInsecure=true`

**Comment by sajeetharan** [comment]
Closing this! please reopen if you face any issue!
